src/Dont/Exception/NonSerialisableObject.php

```diff
@@ -8,6 +8,16 @@
 
 class NonSerialisableObject extends LogicException implements ExceptionInterface
 {
+    private const ERROR_TEMPLATE = <<<'ERROR'
```

Comment:
Explicitly copying to verify. The package provides very little, besides meaningful exception messages, so I just did the additional step of verifying things there.